docs/configuration.md

```diff
@@ -167,6 +167,8 @@ Unless you need to set a non-default value, it is recommended to only populate o
 | `E2E_RELAY_HOST` | ""      | Remote relay url to point to.                                |
 | `DEV_MODE`       | "false" | Flag if relay should operate in developer optimization mode. |
 | `TEST_WS_SERVER` | "false" | Flag config for enable or disable the WS server tests.       |
+| `LOWER_GAS_PRICE_DEVIATION` | 0.80 | Lower value to use when comparing gas prices in the rpc-batch1.spec.ts |
```

Comment:
Q: should we instead make it one single config that gives it a +/- feature. So in this case the deviation is 20% i.e. 0.2

Reply:
Agreed. Done.